src/Commands/BuildCommand.php

```diff
@@ -23,9 +25,18 @@ public function handle(): void
                 echo $output;
             });
 
+        $os = $this->argument('os');
+
+        if ($os == null) {
+            $os = select(
+                label: 'Please select the operating system to build for',
+                options: ['win', 'linux', 'mac', 'all'],
+            );
```

Comment:
Given the discussion in the thread where this started, it would be great if the _current platform_ was the default so the dev doesn't need to type/choose if they're just building to test locally

Reply:
Done.. Please check.